challenges-webapp/src/main/webapp/vue-app/challenges/components/Challenges.vue

```diff
@@ -77,6 +79,22 @@ export default {
     });
     this.$root.$on('challenge-added', this.pushChallenge);
     this.$root.$on('challenge-updated', this.refreshChallenges);
+    const urlPath = document.location.pathname;
+    if (urlPath.includes('challenges/')) {
+      setTimeout(() => {
+        const challengeId = urlPath.split('challenges/')[1].split(/[^0-9]/)[0];
```

Comment:
Could you check if this regex works ? could you try to find a regex that will capture the ID without the need to double split ?